operations/availabilityblocks.md

```diff
@@ -401,57 +354,70 @@ Updates information about the specified [Availability block](#availability-block
+| `TravelAgencyId` | [String update value](_objects.md#string-update-value) | optional | Unique identifier of the `Travel Agency` (or `null` if not updated). |
+| `BookerId` | [String update value](_objects.md#string-update-value) | optional | Unique identifier of the Booker as a creator of an availability block (or `null` if not updated). |
 | `Notes` | [String update value](_objects.md#string-update-value) | optional | Additional notes of the block (or `null` if not updated). |
-| `Budget` | [Currency](accountingitems.md#currency-value) update value | optional | The tentative budget for the total price of reservations (or `null` if not updated). |
+| `Budget` | [CurrencyValueOldUpdateValue](availabilityblocks.md#currencyvalueoldupdatevalue) | optional | The tentative budget for the total price of reservations (or `null` if not updated). |
+| `CancellationReason` | [String update value](_objects.md#string-update-value) | optional | Cancellation reason of the availability block (or `null` if not updated). |
+| `CancellationReasonDetail` | [String update value](_objects.md#string-update-value) | optional | Cancellation reason detail of the availability block (or `null` if not updated). |
 | `RollingReleaseOffset` | [String update value](_objects.md#string-update-value) | optional | Exact offset from the start of availability adjustments to the moment the availability adjustment should be released, in ISO 8601 duration format. Required if `ReleaseStrategy` is set to `RollingRelease`, ignored otherwise. |
 | `ReleasedUtc` | [String update value](_objects.md#string-update-value) | optional | The moment when the block and its availability is released, in UTC timezone ISO 8601 format. Required if `ReleaseStrategy` is set to `FixedRelease`, or used when `ReleaseStrategy` update is unspecified. |
-| `ReleaseStrategy` | [String update value](_objects.md#string-update-value) | optional | The strategy for automatic release of the availability block. If this property is not specified, the release strategy is not updated. |
+| `ReleaseStrategy` | [ReleaseStrategyUpdateValue](availabilityblocks.md#releasestrategyupdatevalue) | optional | The strategy for automatic release of the availability block (or `null` if not updated). |
 
-#### Release strategy
+#### CurrencyValueOldUpdateValue
 
-The strategy for automatic release of the availability block.
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Value` | [Currency value (ver 2018-06-07)](_objects.md#currency-value-ver-2018-06-07) | optional | Total price of the reservation. |
+
+#### ReleaseStrategyUpdateValue
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Value` | [The strategy for automatic release of the availability block.](availabilityblocks.md#the-strategy-for-automatic-release-of-the-availability-block) | required |  |
```

Comment:
I know about the missing description. I mentioned it on backend PR [here](https://github.com/MewsSystems/mews/pull/59262/files#r1761088405)